graviti/manager/branch.py

```diff
@@ -127,6 +131,10 @@ def get(self, name: str) -> Branch:
             self._dataset.name,
             branch=name,
         )
+
+        if hasattr(self._dataset, "HEAD"):
```

Comment:
Why check whether the dataset has `HEAD` here?

Reply:
dataset has no attribute HEAD when we initialize it.